build.psm1

```diff
@@ -414,10 +414,11 @@ Fix steps:
 
     # handle ResGen
     # Heuristic to run ResGen on the fresh machine
-    if ($ResGen -or -not (Test-Path "$PSScriptRoot/src/Microsoft.PowerShell.ConsoleHost/gen")) {
-        Write-Log "Run ResGen (generating C# bindings for resx files)"
-        Start-ResGen
-    }
+    # TODO: remove Start-ResGen after testing ReadyToRun
```

Comment:
Please revert this change.  This is the wrong PR

Reply:
Oops, sorry.